bindings/dart/example/lib/data_provider/data_provider.dart

```diff
@@ -1,7 +1,7 @@
-import 'package:xayn_ai_ffi_dart/package.dart' show FeatureHint, SetupData;
+import 'package:xayn_ai_ffi_dart/package.dart' show SetupData;
 
 /// Prepares and returns the data that is needed to init [`XaynAi`].
-Future<SetupData> getInputData(FeatureHint hint) async =>
+Future<SetupData> getInputData({List<String> features = const []}) async =>
```

Comment:
I think `FeatureHint` can stay, but instead of containing an enum of every possible combinations it only contains the features that can be enabled. We can have `FeatureHint { thread }`, and handle all combinations inside `getInputData`, in the future we have to only add `simd` to the enum.

Reply:
Also can use a `Set` instead of a `List`.
What I meant in the comment of the other PR was only to avoid one enum with a variant for every possible combinations because then we need to handle exponential explosion in the enum, in blue, and our code. With a set of hint we need to handle all the cases in only one place.

Reply:
i see, i'll change it.